cmd/bosun/web/static/js/config.ts

```diff
@@ -223,6 +225,33 @@ bosunControllers.controller('ConfigCtrl', ['$scope', '$http', '$location', '$rou
 	$scope.selectAlert = (alert:string) =>{
 		$scope.selected_alert = alert;
 		$location.search("alert",alert);
+		// Attempt to find `template = foo` in order to set up quick jump between template and alert
+		var searchRegex = new RegExp("^\\s*alert\\s+"+alert, "g");
+		var lines = $scope.config_text.split("\n");
+		$scope.quickJumpTarget = null;
+		for (var i = 0; i < lines.length; i++ ){
+			if(searchRegex.test(lines[i])){
+				for (var j = i+1; j<lines.length; j++){
+					// Close bracket at start of line means end of alert.
+					if (/^\s*\}/m.test(lines[j])){ 
+						return;	
+					}
+					var found = /^\s*template\s*=\s*([\w\-\.\$]+)/m.exec(lines[j]);
+					if(found){
+						$scope.quickJumpTarget = "template " + found[1];
+					}
+				}
+			}
+		}
+	}
+	
+	$scope.quickJump = () =>{
+		var parts = $scope.quickJumpTarget.split(" ");
+		if (parts.length != 2){return;}
+		$scope.scrollTo(parts[0],parts[1]);
+		if(parts[0] == "template" && $scope.selected_alert){
```

Comment:
It could potentially be if there are no alerts in the config, or if the thing has just loaded up. I kinda want to leave the check in just in case. 

We should also clear "quickJumpTarget" at the beginning of this though, just in case a new valid target is not set.